src/main/java/it/gov/pagopa/pu/workflow/wf/receiptclassification/iuf/classification/IufReceiptClassificationWFImpl.java

```diff
@@ -0,0 +1,122 @@
+package it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.classification;
+
+import io.temporal.spring.boot.WorkflowImpl;
+import it.gov.pagopa.payhub.activities.activity.classifications.*;
+import it.gov.pagopa.payhub.activities.dto.classifications.IufClassificationActivityResult;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.dto.IufReceiptClassificationForReportingSignalDTO;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.dto.IufReceiptClassificationForTreasurySignalDTO;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.config.IufReceiptClassificationWfConfig;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.helper.TransferClassificationWfHelperActivity;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+
+import static it.gov.pagopa.pu.workflow.wf.ingestionflow.paymentsreporting.wfingestion.PaymentsReportingIngestionWFImpl.TASK_QUEUE;
+
+/**
+ * Implementation of the IufReceiptClassificationWF interface
+ */
+@Slf4j
+@WorkflowImpl(taskQueues = TASK_QUEUE)
+public class IufReceiptClassificationWFImpl implements IufReceiptClassificationWF, ApplicationContextAware {
```

Comment:
just call it `IufClassification`
```suggestion
public class IufClassificationWFImpl implements IufReceiptClassificationWF, ApplicationContextAware {
```